src/agents/workflows/maingraph.py

```
@@ -70,65 +72,77 @@
                 "passed": False,
-                "feedback": "리뷰 내부 오류(자동 재시도 예정)",
+                "feedback": "Forced review failure for retry simulation.",
                 "suggestions": [],
+                "reasons": ["forced_failure"],
                 "retry_count": current_retry,
             }
         }
+    else:
+        try:
+            patch = run_review(state) or {}
+        except Exception as exc:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)